packages/security/__tests__/CredentialManagerFactory.test.ts

```diff
@@ -155,20 +155,20 @@ describe("CredentialManagerFactory", () => {
             );
         });
 
-        // Note: For some reason using a mock function on logger was not producing the desired results, broke down and mocked stderr
```

Comment:
Is there any reason why this test was commented out in this branch? I see that it must have been in master since the PR is reporting that this is a new change.

Reply:
I missed this change in my reverting of the logging.  Will revert locally and test again.

Reply:
Thanks!